### PR TITLE
[TACHYON-1473] Cleanup of Tachyon server source code

### PR DESCRIPTION
**Pull request by jsimsa**
Addresses: https://tachyon.atlassian.net/browse/TACHYON-1473

### CODE REVIEW

**Comment by jsimsa** [comment]
Jenkins, test this please

File: servers/src/main/java/tachyon/MasterStorageTierAssoc.java [hunk 1]
```diff
@@ -28,6 +28,11 @@ public MasterStorageTierAssoc(TachyonConf conf) {
         Constants.MASTER_TIERED_STORE_GLOBAL_LEVEL_ALIAS_FORMAT);
   }
 
+  /**
+   * Creates a new instance of {@link MasterStorageTierAssoc}.
+   *
+   * @param storageTierAliases storage tier aliases
```
Comment (gpang) [inline servers/src/main/java/tachyon/MasterStorageTierAssoc.java]: maybe add "a list of"?

File: servers/src/main/java/tachyon/master/block/meta/MasterBlockInfo.java [hunk 2]
```diff
@@ -34,6 +34,12 @@
   /** Maps from the worker id to the tier alias the block is on. */
   private final Map<Long, String> mWorkerIdToAlias;
 
+  /**
+   * Creates a new instance of {@link MasterBlockInfo}.
+   *
+   * @param blockId the block id to use
+   * @param length the block length to use
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/block/meta/MasterBlockInfo.java]: add "in bytes"?

File: servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java [hunk 3]
```diff
@@ -62,11 +62,17 @@
...
 
-  /** IDs of blocks the worker contains */
+  /** ids of blocks the worker contains */
   private Set<Long> mBlocks;
-  /** IDs of blocks the worker should remove */
+  /** ids of blocks the worker should remove */
   private Set<Long> mToRemoveBlocks;
 
+  /**
+   * Creates a new isntance of {@link MasterWorkerInfo}.
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java]: isntance -> instance

File: servers/src/main/java/tachyon/master/file/meta/Inode.java [hunk 4]
```diff
@@ -46,41 +49,73 @@ public Builder() {
...
+     */
     public T setParentId(long parentId) {
       mParentId = parentId;
       return getThis();
     }
 
-    public T setPersistenceState(PersistenceState filePersistenceState) {
-      mPersistenceState = filePersistenceState;
+    /**
+     * @param persistenceState the persistence state to use
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/file/meta/Inode.java]: use the "{@link PersistenceState}" notation?

File: servers/src/main/java/tachyon/master/file/meta/Inode.java [hunk 5]
```diff
@@ -46,41 +49,73 @@ public Builder() {
...
+     * @return the builder
+     */
+    public T setPersistenceState(PersistenceState persistenceState) {
+      mPersistenceState = persistenceState;
       return getThis();
     }
 
+    /**
+     * @param ps the permission status to use
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/file/meta/Inode.java]: use the "{@link PermissionStatus}" notation?

File: servers/src/main/java/tachyon/metrics/MetricsConfig.java [hunk 6]
```diff
@@ -94,9 +94,9 @@ public Properties getInstanceProperties(String inst) {
   }
 
   /**
-   * Gets the propertyCategories, used by unit tests only.
+   * Gets the property categories, used by unit tests only.
    *
-   * @return a Map that maps from instance name to its properties
+   * @return a {@code Map} that maps from instance name to its properties
```
Comment (gpang) [inline servers/src/main/java/tachyon/metrics/MetricsConfig.java]: should this use @code or @link?

File: servers/src/main/java/tachyon/metrics/MetricsConfig.java [hunk 7]
```diff
@@ -159,7 +159,7 @@ private void setDefaultProperties() {
    *
    * @param prop the original properties
    * @param regex specifies the prefix and suffix pattern
-   * @return a Map maps from the prefix to its properties
+   * @return a {@code Map} maps from the prefix to its properties
```
Comment (gpang) [inline servers/src/main/java/tachyon/metrics/MetricsConfig.java]: should this use @code or @link?

File: servers/src/main/java/tachyon/metrics/sink/CsvSink.java [hunk 8]
```diff
@@ -41,7 +41,7 @@
   private Properties mProperties;
 
   /**
-   * Creates a CsvSink with a {@link Properties} and {@link MetricRegistry}.
+   * Creates a new {@ CsvSink} with a {@link Properties} and {@link MetricRegistry}.
```
Comment (gpang) [inline servers/src/main/java/tachyon/metrics/sink/CsvSink.java]: {@ CsvSink} -> {@link CsvSink}

File: servers/src/main/java/tachyon/web/WebInterfaceGeneralServlet.java [hunk 9]
```diff
@@ -113,9 +136,9 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
   }
 
   /**
-   * List the {@link StorageTierInfo} objects of each storage level(alias).
+   * List the {@link StorageTierInfo} objects of each storage level alias.
```
Comment (gpang) [inline servers/src/main/java/tachyon/web/WebInterfaceGeneralServlet.java]: List -> Lists

File: servers/src/main/java/tachyon/worker/block/BlockDataManager.java [hunk 10]
```diff
@@ -385,21 +385,28 @@ public void requestSpace(long sessionId, long blockId, long additionalBytes)
...
+   * @throws BlockDoesNotExistException if lock id cannot be found
    */
   public void unlockBlock(long lockId) throws BlockDoesNotExistException {
     mBlockStore.unlockBlock(lockId);
   }
 
-  // TODO(calvin): Remove when lock and reads are separate operations.
+  /**
+   * Releases the lock with the specified session and block id.
+   * TODO(calvin): Remove when lock and reads are separate operations.
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockDataManager.java]: should TODO's be in the javadoc?

File: servers/src/main/java/tachyon/worker/block/BlockLockManager.java [hunk 11]
```diff
@@ -135,6 +138,14 @@ public void unlockBlock(long lockId) throws BlockDoesNotExistException {
     lock.unlock();
   }
 
+  /**
+   * Releases the lock with the specified session and block id.
+   * TODO(calvin): Remove when lock and reads are separate operations.
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockLockManager.java]: Where did this TODO come from?

File: servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java [hunk 12]
```diff
@@ -59,8 +59,8 @@
   private Map<String, StorageTierView> mAliasToTierViews = new HashMap<String, StorageTierView>();
 
   /**
-   * Constructor of BlockMatadataManagerView. Now we always creating a new view before freespace.
-   * TODO(qifan): Incrementally update the view.
+   * Creates a new instance of {@link BlockMetadataManagerView}. Now we always create a new view
+   * before freespace. TODO(qifan): Incrementally update the view.
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java]: Do we want TODO's in javadoc?

File: servers/src/main/java/tachyon/worker/block/meta/TempBlockMeta.java [hunk 13]
```diff
@@ -23,10 +23,10 @@
   private long mTempBlockSize;
 
   /**
-   * Constructs the metadata of a temp block.
+   * Creates a new instace of {@link TempBlockMeta}.
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/meta/TempBlockMeta.java]: instace -> instance

File: servers/src/main/java/tachyon/worker/file/FileDataManager.java [hunk 14]
```diff
@@ -136,6 +141,11 @@ private String prepareUfsFilePath(long fileId) throws IOException {
     return dstPath;
   }
 
+  /**
+   * Returns a a list of file to persist, removing them from the internal queue.
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/file/FileDataManager.java]: duplicate "a"

**Comment by gpang** [comment]
Thanks for all the cleanup! I left some comments.

File: servers/src/main/java/tachyon/MasterStorageTierAssoc.java (hunk 1 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/MasterStorageTierAssoc.java]: done

File: servers/src/main/java/tachyon/master/block/meta/MasterBlockInfo.java (hunk 2 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/block/meta/MasterBlockInfo.java]: done

File: servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java (hunk 3 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java]: done

File: servers/src/main/java/tachyon/master/file/meta/Inode.java (hunk 4 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/file/meta/Inode.java]: done

File: servers/src/main/java/tachyon/master/file/meta/Inode.java (hunk 5 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/file/meta/Inode.java]: done

File: servers/src/main/java/tachyon/metrics/MetricsConfig.java (hunk 6 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/metrics/MetricsConfig.java]: I use `code` for classes that are external to Tachyon code base

File: servers/src/main/java/tachyon/metrics/MetricsConfig.java (hunk 7 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/metrics/MetricsConfig.java]: ditto

File: servers/src/main/java/tachyon/metrics/sink/CsvSink.java (hunk 8 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/metrics/sink/CsvSink.java]: done

File: servers/src/main/java/tachyon/web/WebInterfaceGeneralServlet.java (hunk 9 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/web/WebInterfaceGeneralServlet.java]: done

File: servers/src/main/java/tachyon/worker/block/BlockLockManager.java (hunk 11 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/worker/block/BlockLockManager.java]: done

File: servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java (hunk 12 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java]: done

File: servers/src/main/java/tachyon/worker/block/meta/TempBlockMeta.java (hunk 13 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/worker/block/meta/TempBlockMeta.java]: done

File: servers/src/main/java/tachyon/worker/file/FileDataManager.java (hunk 14 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/worker/file/FileDataManager.java]: done

File: servers/src/main/java/tachyon/worker/block/BlockDataManager.java (hunk 10 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/worker/block/BlockDataManager.java]: done

**Comment by jsimsa** [comment]
@gpang addressed your comments, PTAL

**Comment by yupeng9** [comment]
LGTM.Thanks for the cleanup!

**Comment by yupeng9** [comment]
Please resolve the conflicts

**Comment by jsimsa** [comment]
done

**Comment by jsimsa** [comment]
@yupeng9 @gpang do you have any additional comments? if not, I would like to merge this PR to avoid having to resolve additional merge conflicts

**Comment by gpang** [comment]
LGTM

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**